### PR TITLE
Add specs for group activity logs

### PR DESCRIPTION
**Pull request by gsarantid**
Task: https://app.shortcut.com/tiledb-inc/story/45481/define-specifications-for-group-activity-logs

### CODE REVIEW

File: openapi-v2.yaml [hunk 1]
```diff
@@ -1983,6 +1983,79 @@ definitions:
...
+    type: object
+    properties:
+      activitylogs:
+        description: Array of GroupActivityLog
+        type: array
+        x-omitempty: false
+        items:
+          $ref: "#/definitions/GroupActivityLog"
+      pagination_metadata:
+        x-omitempty: false
```
Comment (gsarantid) [inline openapi-v2.yaml:1997]: @alekshs this should be false or true?

File: openapi-v2.yaml [hunk 2]
```diff
@@ -2626,6 +2699,64 @@ paths:
...
+      - name: per_page
+        in: query
+        description: pagination limit
+        type: integer
+        required: false
+    get:
+      description: get group activity logs
+      tags:
+        - group
+      operationId: groupActivityLog
```
Comment (alekshs) [inline openapi-v2.yaml:2747]: I would name this listGroupActivity or something

File: openapi-v2.yaml [hunk 3]
```diff
@@ -1983,6 +1983,79 @@ definitions:
...
         $ref: "#/definitions/PaginationMetadata"
 
+  GroupActivityLogResponse:
+    description: Object containing activity logs of a group along with the pagination metadata
+    type: object
+    properties:
+      activitylogs:
+        description: Array of GroupActivityLog
+        type: array
+        x-omitempty: false
```
[the remark this answers is not in the document]
Comment (alekshs) [reply openapi-v2.yaml:1993]: It is a better practice to set this to false and always return a slice in the response (even empty) in order to have a single data type returned to the clients.

File: openapi-v2.yaml (hunk 1 above)
Comment (alekshs) [reply openapi-v2.yaml:1997]: There should always be a metadata object i guess

File: openapi-v2.yaml (hunk 2 above)
Comment (gsarantid) [reply openapi-v2.yaml]: Fixed

**Review by alekshs — APPROVED** [review]
LGTM

**Comment by gsarantid** [comment]
@sammysaglam @snagles wanna have an extra look? Thanks

File: openapi-v2.yaml [hunk 4]
```diff
@@ -2000,6 +2000,79 @@ definitions:
         x-omitempty: false
         $ref: "#/definitions/PaginationMetadata"
 
+  GroupActivityLogResponse:
+    description: Object containing activity logs of a group along with the pagination metadata
+    type: object
+    properties:
+      activitylogs:
```
[the remark this answers is not in the document]
Comment (gsarantid) [reply openapi-v2.yaml]: Fixed!

File: openapi-v2.yaml [hunk 5]
```diff
@@ -2650,6 +2723,64 @@ paths:
...
+        required: false
+      - name: per_page
+        in: query
+        description: pagination limit
+        type: integer
+        required: false
+    get:
+      description: get group activity logs
+      tags:
+        - group
```
Comment (alekshs) [inline openapi-v2.yaml:2770]: This should be `groups`

**Review by alekshs — CHANGES_REQUESTED** [review]

File: openapi-v2.yaml (hunk 5 above)
Comment (gsarantid) [reply openapi-v2.yaml]: Fixed

**Review by alekshs — APPROVED** [review]

**Pull request merged**